build.sbt

```diff
@@ -44,7 +47,11 @@ libraryDependencies ++= Seq(
   "com.fasterxml.jackson.module" %% "jackson-module-scala"          % "2.9.6",
   "com.chuusai"                  %% "shapeless"                     % "2.3.3",
   "org.typelevel"                %% "cats-core"                     % catsVersion,
-  "org.typelevel"                %% "mouse"                         % "0.20",
+  "org.typelevel"                %% "cats-effect"                   % catsVersion,
```

Comment:
I wouldn't use the same version val (cats core and effect can have different veresions)

Reply:
ouch... I thought they were aligning

Reply:
Just in the majors, I believe, and that's sort of by accident...